### PR TITLE
Feature/new homepage

### PR DESCRIPTION
**Pull request by nishilfaldu**
# Notes for blog or docs authors:

-   The `image: ` field in frontmatter of blogs should always be a `png` (`svg` will not work for social previews)
-   You must also duplicate the asset used in `image :` inside of the [public folder](https://github.com/tembo-io/website/tree/main/public)
-   Please write a `description: ` in the frontmatter of any new blog or doc
-   Read more [here](https://github.com/tembo-io/website?tab=readme-ov-file#writing-a-blog-post-%EF%B8%8F)

Link to the ticket: https://linear.app/tembo/issue/CLOUD-853/website-new-homepage

### CODE REVIEW

**Comment by aishwaryaborkar** [comment]
We probably want to have the quotes and at the very least, have Postgres capitalized as per design: 
Screenshot 2024-06-18 at 4 15 16 PM

**Comment by aishwaryaborkar** [comment]
Screenshot 2024-06-18 at 4 17 21 PM
Screenshot 2024-06-18 at 4 17 28 PM

The titles are getting cut off/covered up on mobile. Apologies if you were already aware of this - just saw that this was ready for review and trying to get more familiar with website repo.

File: src/components/StackHeroCards.astro [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import MongoAlternativeStackElephant from '../images/mongoalternative-stack-elephant.svg';
+import GeospatialStackElephant from '../images/geospatial-stack-elephant.svg';
+import DataWareHouseStackElephant from '../images/datawarehouse-stack-elephant.svg';
+import MQStackElephant from '../images/mq-stack-elephant.svg';
+import StandardStackElephant from '../images/standard-stack-elephant.svg';
+import TimeseriesStackElephant from '../images/timeseries-stack-elephant.svg';
+import OLTPStackElephant from '../images/oltp-stack-elephant.svg';
+import OLAPStackElephant from '../images/olap-stack-elephant.svg';
+
+const stackHeroCardDetails = [
```
Comment (aishwaryaborkar) [inline src/components/StackHeroCards.astro:17]: The next few comments going to be super nitpicky about the copy for all the cards but given that this is the marketing site, I assume it's important for consistency. I'm also assuming this just got copied/pasted from Figma.

File: src/components/StackHeroCards.astro [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+	{
+		title: 'DataWarehouse',
+		description:
+			'Extract, Transform and Load data from external sources. Build centralized datastore for analytical and tactical queries',
+		image: DataWareHouseStackElephant,
```
Comment (aishwaryaborkar) [inline src/components/StackHeroCards.astro:22]: Build *a* centralized datastore

Add a period at the end

File: src/components/StackHeroCards.astro [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+	},
+	{
+		title: 'MessageQueue',
+		description:
+			'An alternative to AWS SQS and Redis RSMQ, on Postgres. Interface to Postgres queues using SQL, REST API, and many client libraries including Python, Rust, and Go.',
```
Comment (aishwaryaborkar) [inline src/components/StackHeroCards.astro:27]: No need for comma after Redis RSMQ

**Comment by DarrenBaldwin07** [comment]
> We probably want to have the quotes and at the very least, have Postgres capitalized as per design: Screenshot 2024-06-18 at 4 15 16 PM

@aishwaryaborkar FYI this isnt ready to get merged. Samay will be doing a full pass on all the copy before merging (everything is just placeholder right now). The top hero is not finished yet and will be animate also the customer quotes will be a carrousel like on the pricing page

**Review by aishwaryaborkar — APPROVED** [review]

**Pull request merged**